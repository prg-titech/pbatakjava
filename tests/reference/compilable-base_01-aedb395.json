{
    "basename": "compilable-base_01-aedb395",
    "cmd": "{infile}",
    "infile": "tests/base_01/build.sh",
    "infile_hash": "3ecade6a40d974cd0d01860402f3dd4b4eadf78d6fcdcde5dfc2e5d9",
    "stdout": "compilable-base_01-aedb395.stdout",
    "stdout_hash": "d07145110ea7282446e86887e50d774107e87df7963bdc152c490f50",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}