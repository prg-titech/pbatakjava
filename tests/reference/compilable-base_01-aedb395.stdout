
public interface C {
}

public interface C_Factory {
}

public class C_v1 implements C {
}

public class C_v1_Factory implements C_Factory {
}

public class C_v2 implements C {
}

public class C_v2_Factory implements C_Factory {
}
